lib/apartment.rb

```diff
@@ -3,6 +3,12 @@ class Apartment <ActiveRecord::Base
   has_and_belongs_to_many(:users)
 
 
+#zip codes:
+#North Portland 97217
+#Northeast Portland 97213
+#Northwest Portland 97229
+#Southeast Portland 97206
```

Comment:
What about 97214? :(

Reply:
So, we selected zip codes that will have the best results per quadrant if you apply (in most cases) a 3 mile radius, 97214 is indeed a SE quadrant zip, but one further out a bit would help to prevent some crossover into other quadrants while capturing much of the outer zips as well.

Reply:
Ooohhh that make sense. I didn't take into account the encompassing radius from the 97206 center.  Clever clever